mne/viz/utils.py

```diff
@@ -985,32 +985,39 @@ def add_background_image(fig, im, set_ratios=None):
 
     Parameters
     ----------
-    fig: plt.figure
+    fig : plt.figure
         The figure you wish to add a bg image to.
-    im: ndarray
+    im : ndarray
         A numpy array that works with a call to
         plt.imshow(im). This will be plotted
         as the background of the figure.
-    set_ratios: None | str
+    remove_bg : bool
```

Comment:
yes please, only leave it in if it's necessary (and clear when/why it is necessary)